home/blocks.py

```diff
@@ -13,25 +14,40 @@ def render_basic(self, value, context=None):
         if not value:
             return ''
 
+        video_not_supported_text = _("Your browser does not support the video tag.")
+        audio_not_supported_text = _("Your browser does not support the audio element.")
+        # Translators: Part of this message (between %(start_link)s and %(end_link)s ) is a clickable download link
+        download_video_text = _('If you cannot view the above video, perhaps would'
```

Comment:
It's normally better to use f strings for creating templated strings. It's more readable

Reply:
The last paragraph from [here](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#standard-translation) states:

"Since string extraction is done by the xgettext command, only syntaxes supported by gettext are supported by Django. In particular, Python f-strings are not yet supported by xgettext, and JavaScript template strings need gettext 0.21+."

Thus I will leave this as it is.